tests/phpunit/tests/option/networkOption.php

```diff
@@ -697,8 +697,13 @@ public function data_stored_as_empty_string() {
 	 * @covers ::update_network_option
 	 */
 	public function test_update_network_option_with_pre_filter_adds_missing_option() {
+
+		if ( is_multisite() ) {
```

Comment:
For all of these, rather than skipping based on whether the tests are running with a multisite configuration or not, and having duplicate tests, can we do something like this instead?

```
$filter = is_multisite() ? 'pre_site_option_foo' : 'pre_option_foo';
```

Reply:
Done in [219f839](https://github.com/WordPress/wordpress-develop/pull/5434/commits/219f839f63ebd365946988b2bea4571eaa4dcc55). Test methods were renamed and moved for better grouping, but no tests were fundamentally changed.

Reply:
Done in [219f839](https://github.com/WordPress/wordpress-develop/pull/5434/commits/219f839f63ebd365946988b2bea4571eaa4dcc55). Test methods were renamed and moved for better grouping, but no tests were fundamentally changed.